maxim/tests/test_openai.py

```
@@ -158,54 +158,106 @@ def test_chat_completions_using_wrapper(self):
         client = MaximOpenAIClient(
             OpenAI(api_key=openaiApiKey), logger=self.logger
         )
         response = client.chat.completions.create(
-            messages=[{"role": "user", "content": "Explain how AI works"}],
+            messages=[{"role": "user", "content": "Explain how Nascar works"}],
             model="gpt-4o",
             max_tokens=1000,
             stream=True,
+            stream_options={"include_usage": True},
         )
         for event in response:
```

Ruff (0.12.2)

Loop control variable `event` not used within loop body

Rename unused `event` to `_event`

(B007)